napari/_qt/menus/plugins_menu.py

```diff
@@ -34,8 +37,9 @@ def __init__(self, window: 'Window') -> None:
 
     def _build(self, event=None):
         self.clear()
-        action = self.addAction(trans._("Plugin Manager"))
-        action.triggered.connect(self._show_plugin_install_dialog)
+        if self._plugin_manager_dialog_cls() is not None:
+            action = self.addAction(trans._("Plugin Manager"))
+            action.triggered.connect(self._show_plugin_install_dialog)
         action = self.addAction(trans._("Plugin Errors..."))
         action.setStatusTip(
```

Comment:
I am not familiar with this but would it be possible to have a plugin installed without the plugin manager? If not, should this part be within the `if` as well? cc @jaimergp

Reply:
Maybe you could install a plugin, then uninstall the plugin manager?

Reply:
Ah I see the intent, it's just because `QtPluginDialog` was moved but `QtPluginErrReporter` was not. But the question may still be of interest, does it make sense to have a plugin error menu item in the case when plugin manager is not installed?

Reply:
Nevermind, I see it just allows installation via GUI. Sorry for noise.